### PR TITLE
check functionality

### PR DESCRIPTION
**Pull request by karan14verma**

### CODE REVIEW

**Comment by AsharShoaib** [comment]
Hmmm

**Comment by NJWewala** [comment]
> _No description provided._

File: src/Item/Components/ListItem.tsx [hunk 1]
```diff
@@ -11,17 +10,32 @@ interface IState{
 }
 
 class ListItem extends React.Component<IProps, IState> {
-    public render() {
+    state={
+        isChecked : true,
```
Comment (NJWewala) [inline src/Item/Components/ListItem.tsx]: isChecked : false and not true cause the webpage reloads with already strikethrough text

Comment (karan14verma) [reply src/Item/Components/ListItem.tsx]: yooo thanks nice catch

**Pull request merged**